README.md

```diff
@@ -20,3 +20,10 @@ in. The end of this project will include a baseline for STEM Ecosystem to grow
 into the future with their users on a scalable platform to help continue the
 advancement of students in science, technology, engineering, and mathematics.
 
+## [Project-oriented Risk List](https://github.com/MLHale/CYBR4580-8950/blob/master/projects/milestone1.md#risk-list)
+| Risk Name (value) | Impact | Likelihood | Description |
+|-------------------------------------------------------|--------|------------|:---------------------------------------------------------------------------------------------------------------------------------------:|
+| Misunderstanding the project requirements (75) | 9 | 8 | When the requirements are not clearly understood by team. |
+| Lack of programming language  (50)        | 8 | 7 | If any high level issues come across while developing web page will slow down the project progress. |
```

Comment:
I would change this description to something like: `Any high level issues that come up while developing the web page will slow down the project progress.`